### ISSUE TITLE
Automated build/push with GitHub Actions

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/drupal-container#8]
See how it's being done in the upstream `geerlingguy/php-apache` container image build process. Basically add the docker login details as protected env vars in Travis CI, then edit the `.travis.yml` file to do an automated push once the build is complete on the master branch.

**Comment by geerlingguy** [comment]
Updating since Travis CI is no more. See #29.

**Comment by geerlingguy** [comment]
Hmm... all builds succeed besides arm64. I'll leave that out and open a new follow-up issue for it.

**Comment by geerlingguy** [comment]
Closing with the latest commit.
